src/contexts/course-mode/entities/multiple-choice-question.ts

```diff
@@ -0,0 +1,32 @@
+import { Answer, Question, QuestionId, QuestionServices, ValidationResult } from './question';
```

Comment:
do we want to generate choice questions too? i guess in theory open-ended questions work best for active recall

Reply:
cool, let's do that then